formcentral/config.go

```diff
@@ -0,0 +1,13 @@
+package main
+
+import (
+	"os"
+)
+
+var (
+	PostgresDb       = os.Getenv("POSTGRES_DB")
```

Comment:
The app already contains this config information: https://github.com/vlab-research/fly/blob/main/formcentral/server.go#L75

We currently use this library to parse env vars and validate them: https://github.com/vlab-research/fly/blob/main/formcentral/server.go#L8

Reply:
Sounds good, removed, https://github.com/vlab-research/fly/pull/23/commits/7c27fba851a01f77246a3c94208f92292c8753a4.